src/pages/en/index.mdx

```diff
@@ -77,31 +79,50 @@ import {
 </ContentSection>
 
 <ContentSection variant="white" id="what-is-a-tolocar">
-  <HeadlineUnderlined large variant={5} className="container-width mt-8 sm:mt-20">What is a Tolocar?</HeadlineUnderlined>
-  <div class="container-width flex lg:justify-end mt-8 sm:mt-16 mb-16">
-    <div class="flex flex-col lg:w-10/12 ">
-      <div class="flex flex-col lg:flex-row custom-prose text-lg leading-6 gap-8 mb-12 lg:mb-20">
-        <div class="flex-1">
+  <HeadlineUnderlined large variant={6} className="container-width mt-8 sm:mt-20">What is a Tolocar?</HeadlineUnderlined>
+  <div class="container-width flex lg:justify-start mt-8 sm:mt-16 mb-8">
+    <div class="flex flex-col">
+      <div class="flex flex-col lg:flex-row custom-prose text-lg leading-6 gap-8 mb-6 lg:mb-10">
+        <div class="flex-[40%]">
```

Comment:
For "round" percentage values, you can use integer values. E.g. instead of `flex-[40%]` you can use `flex-[4]` and instead of `flex-[60%]` you can use `flex-[6]`.

But that's a question of taste. Functionally, both works fine.

Reply:
I'll keep that in mind! Thanks for the tip! 🙏